README.md

```diff
@@ -207,6 +208,26 @@ The following documentation is available:
 * [Release Notes](https://github.com/minishift/minishift/releases)
 * [Developing Minishift](./docs/developing.md)
 
+<a name="using-minishift-with-a-proxy"></a>
+## Using Minishift with a HTTP/HTTPS Proxy
+
+If you are behind a HTTP/HTTPS proxy, you may need to supply proxy options.
+To do this, pass the required flags during `minishift start`.
+
+For example:
+
+```shell
+$ minishift start --http-proxy http://YOURPROXY:PORT \
+                 --https-proxy https://YOURPROXY:PORT
+```
+
+ In an authenticated proxy environment, the `proxy_user` and `proxy_password` should be part of proxy URI.
+ 
+```shell
+ $ minishift start --http-proxy http://<proxy_username>:<proxy_password>@YOURPROXY:PORT \
+                  --https-proxy https://<proxy_username>:<proxy_password>YOURPROXY:PORT
+```
+
```

Comment:
Looking good so far, but I was hoping for some additional context on what this does for the user. Aka, the various parts which gets configured for proxies.

Might be worth to have a look at https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#using-a-proxy and maybe even reference it.